src/__tests__/unit/lib/boavizta/index.test.ts

```diff
@@ -188,6 +247,115 @@ describe('lib/boavizta: ', () => {
           expect(error).toBeInstanceOf(InputValidationError);
         }
       });
+
+      it('throws an error when `provider` is wrong.', async () => {
+        expect.assertions(1);
+
+        try {
+          await output.execute([
+            {
+              timestamp: '2021-01-01T00:00:00Z',
+              duration: 15,
+              'cpu/utilization': 34,
+              'instance-type': 't2.micro',
+              country: 'USA',
+              provider: 'aws1',
+            },
+          ]);
+        } catch (error) {
+          expect(error).toBeInstanceOf(UnsupportedValueError);
```

Comment:
please also assert the error message if possible

Reply:
done